src/darkmode.css

```diff
@@ -1,107 +1,72 @@
-/* darkmode.css */
 
-/* Define dark mode styles */
-/* .app.dark {
-    background: verydarkblue;
-    color: white;
-  }
-  
-  .app.dark header {
-    background: darkblue;
-    box-shadow: 0 3px 4px shadowHeader;
-  }
-  
-  .app.dark .contents {
-    background: verydarkblue;
-  }
-  
-  .app.dark .card,
-  .app.dark .searchBox,
-  .app.dark .selectTitle,
-  .app.dark .selectList {
-    background:darkblue;
-    box-shadow: 3px 3px 4px shadowInput;
-  }
-  
-  .app.dark .backBtn,
-  .app.dark .borderCountry {
-    background: darkblue;
-    box-shadow: 0 0 4px 1px shadowInput;
-    color: white;
-  }
-  
-  .app.dark .selectList li {
-    background: darkblue;
-    color: white;
-  } */
-  
-  /* Include other dark mode styles as needed */
-  
-
-  /* darkmode.css */
-
-/* Define dark mode styles */
 .App.dark {
-    background: rgb(23, 23, 23);
-    color: white;
+  background: hsl(207, 26%, 17%);
+  color: white;
 }
 
+.App.dark .contents {
+  background: hsl(207, 26%, 17%);
+}
 .App.dark header {
```

Comment:
a better approach is to make just two classes for dark mode. one for background and one for elements(header, buttons and cards are all same color)